tests/upgrade/basic/task.yaml

```diff
@@ -45,8 +45,11 @@ execute: |
     snap list | grep test-snapd-tools
     test-snapd-tools.echo Hello | grep Hello
     test-snapd-tools.env | grep SNAP_NAME=test-snapd-tools
-    echo Hello > /var/tmp/myevil.txt
-    test-snapd-tools.cat /var/tmp/myevil.txt && exit 1 || true
+
+    if [ -e /sys/kernel/security/apparmor ]; then
```

Comment:
Maybe a small comment like: `only test if confinement works if we actually have apparmor available` ?